Resources/Private/CodeTemplates/Extbase/Documentation.tmpl/Settings.cfgt

```diff
@@ -80,7 +78,7 @@ h2document    = https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us
 # t3coreapi     = https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/
 # t3editors     = https://docs.typo3.org/m/typo3/tutorial-editors/master/en-us/
 # t3extbasebook = https://docs.typo3.org/m/typo3/book-extbasefluid/master/en-us/Index.html
-# t3install     = https://docs.typo3.org/m/typo3/guide-installation/master/en-us/
+#t3install     = https://docs.typo3.org/m/typo3/guide-installation/master/en-us/
```

Comment:
In the original version, this line is active. Is it on purpose commented?

Reply:
Yes, because usually you don't need a reference to the general TYPO3 guide

Reply:
Ok